Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -672,13 +679,54 @@ ModelExperimentalSceneGraph.prototype.updateJointMatrices = function () {
   }
 };
 
-function forEachRuntimePrimitive(sceneGraph, callback) {
-  for (let i = 0; i < sceneGraph._runtimeNodes.length; i++) {
-    const runtimeNode = sceneGraph._runtimeNodes[i];
-    for (let j = 0; j < runtimeNode.runtimePrimitives.length; j++) {
-      const runtimePrimitive = runtimeNode.runtimePrimitives[j];
-      callback(runtimePrimitive);
-    }
+/**
+ * Recursively traverse through the runtime nodes in the scene graph
+ * using depth-first post-order traversal to perform a callback on
```

Comment:
Just noticed this, I think this wording is clearer since "post-order" is modifying "depth-first traversal"

```suggestion
 * using a post-order depth-first traversal to perform a callback on
```